src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java

```diff
@@ -85,6 +85,11 @@
     private static final String NOT_EXISTING_FILE = "target/not_existing.xml";
     private static final String FAILURE_PROPERTY_VALUE = "myValue";
 
+    @Override
+    protected String getPackageLocation() throws IOException {
+        return "com/puppycrawl/tools/checkstyle";
```

Comment:
@rnveach There is an issue here.
The moment I change the directory, I am getting errors because few of the xml files that `CheckstyleAntTaskTest` uses lie outside the folder.
They are in `com/puppycrawl/tools/checkstyle/` and not `com/puppycrawl/tools/checkstyle/ant`
Either we need to work another PR where move them into `CheckstyleAntTaskTest` or I might have to keep the directory as is.